server/src/main/java/com/lunchtime/controllers/EventController.java

```diff
@@ -0,0 +1,33 @@
+package com.lunchtime.controllers;
+
+import com.lunchtime.entity.Event;
+import com.lunchtime.service.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/events")
+public class EventController {
+
+    @Autowired
+    EventService eventService;
```

Comment:
Use constructor injecting instead of `@Autowired` annotation:

```
private final EventService eventService;

public EventController(EventService eventService) {
         this.eventService = eventService;
}
```

The same as line 15 [here](https://github.com/Social-projects-Rivne/Lunch_Time/pull/19/files?file-filters%5B%5D=.css&file-filters%5B%5D=.java&file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-0607fa89e7d13943be76841f7d4b65c9)